### ISSUE TITLE
Single PC, multiple monitors

### ISSUE THREAD

**Issue by tellzey1** [fuel9/DisplayMonkey#153]
Is there a way to get a single PC with 2 or more monitors to register 2 or more Display instances so that each browser instance can display a different Canvas?  
I have 18 50" displays on a video wall with a PC controlling 2 to 4 displays each.  I cannot setup 18 PCs to run the 18 displays.  Any suggestions??

**Comment by fluxededge** [comment]
Yes, if you go to "Currently Displayed" and click the name of your Display to preview it. It will come up with a link. Each Canvas has a specified link with a number at the end. Example: **getCanvas.aspx?display=48** This number identifies each separate Canvas. So you could run each Chrome instance with a different link assigned to each monitor.
